docs/_includes/generated/task_manager_configuration.html

```diff
@@ -0,0 +1,146 @@
+<table class="table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Key</th>
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 65%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>task.cancellation.interval</h5></td>
```

Comment:
The description for those two should probably be carried over from the old doc

Reply:
there's a description for cancellation.interval, which other one are you referring to?

Reply:
`cancellation.timeout`, I think I was referring to